ads/_config.js

```diff
@@ -762,6 +762,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'promoteiq': {},
```

Comment:
could you try if you can implement renderStart API?

you will need to call `context.renderStart` when the ad response is back . for case of no fill, you will call `context.noContentAvailable`

Reply:
Unfortunately, we add the ad tag to the DOM, but it does not trigger an event before it is rendered. We do return an id of the call to our ad server, but an id is returned whether or not an ad is rendered.

Reply:
@torch2424 Unfortunately, the way our tag works does not really allow us to implement the `context.renderStart`and `context.noContentAvailable` APIs. We give our clients a lot of control on how they want to to handle rendering on their site. This means that we give them the control on how to handle responses from PromoteIQ (whether it is an empty response or not).